### PR TITLE
Rename odo.

### PR DESCRIPTION
**Pull request by yhontyk**
- `odo` has been renamed - "OpenShift Do" is no longer used.
- the direct request from the stakeholders - not to mention OpenShift specifically if it can be avoided and not to advertise odo as an OpenShift tool (it works both for OpenShift and Kubernetes), I renamed everything to from `OpenShift Developer CLI` to just `Developer CLI` (as per team requests) and removed OCP mentions where it can be avoided
- got rid of an odo verison attribute since it is redundant

Applies only to 4.4 and 4.5 branches as there will probably be conflicts if we cherry-pick to earlier ones (some file structure changed from release to release). I'll submit PRs against 4.3 and 4.2 separately when this one is reviewed and merged.

### CODE REVIEW

File: cli_reference/developer_cli_odo/understanding-odo.adoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 = Understanding OpenShift Do
-include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/developer-cli-odo-attributes.adoc[]
 include::modules/common-attributes.adoc[]
 :context: understanding-odo
 toc::[]
 
-{odo-title-full} (`{odo-title}`) is a fast and easy-to-use CLI tool for creating applications on {product-title}. `{odo-title}` allows developers to concentrate on creating applications without the need to administrate an {product-title} cluster itself.
-Creating deployment configurations, build configurations, service routes and other {product-title} elements are all automated by `{odo-title}`.
+`{odo-title}` is a fast and easy-to-use CLI tool for creating applications on {product-title} and Kubernetes. `{odo-title}` allows developers to concentrate on creating applications without the need to administrate a cluster itself.
```
Comment (ahardin-rh) [inline cli_reference/developer_cli_odo/understanding-odo.adoc:8]: s/easy-to-use/easy to use 
I think this sounds like marketing speak, so I suggest removing "fast and easy-to-use" altogether.

s/administrate/administer

File: cli_reference/developer_cli_odo/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+toc::[]
+
+After you push the `init` image to a cluster with a mirrored registry, you must mirror a supported builder image for your application with the `oc` tool, overwrite the mirror registry using the environment variable, and then create your component.
+
+.Prerequisites
+
+* Install `oc` on the client operating system.
+* xref:../../../cli_reference/developer_cli_odo/installing-odo.adoc#installing-odo-on-linux[Install `{odo-title}`] on the client operating system.
+* Access to an restricted cluster with a configured internal registry or a mirror registry.
+* xref:../../../cli_reference/developer_cli_odo/using_odo_in_a_restricted_environment/pushing-the-odo-init-image-to-the-restricted-cluster-registry.html#pushing-the-odo-init-image-to-a-mirror-registry_pushing-the-odo-init-image-to-the-restricted-cluster-registry[Push the `odo` init image to your cluster registry].
```
Comment (ahardin-rh) [inline cli_reference/developer_cli_odo/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster.adoc:15]: s/.html/.adoc

**Comment by ahardin-rh** [comment]
@boczkowska Just a few minor comments from me. Thanks!

**Comment by boczkowska** [comment]
@ahardin-rh fixed everything! Thank you for the review!
Could we merge it tomorrow when @vikram-redhat configures redirects for old URLs?

**Comment by vikram-redhat** [comment]
> @ahardin-rh fixed everything! Thank you for the review!
> Could we merge it tomorrow when @vikram-redhat configures redirects for old URLs?

I will merge it today when I have gone through the redircts.

**Pull request merged**